src/AzureFunctions.PowerShell.OpenTelemetry.SDK/Traces/NewTracerBuilder.cs

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Management.Automation;
+using OpenTelemetryEngine.Traces;
+
+namespace AzureFunctions.PowerShell.OpenTelemetry.SDK
+{
+    /// <summary>
+    /// Cmdlet that sets up the OpenTelemetry TracerBuilder
+    /// This Logger is configured with an OTLP exporter that will send traces to an OTLP endpoint defined by the user
+    /// See https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "TracerBuilder")]
```

Comment:
I would normally expect a cmdlet that starts with "New" to output a newly created object, which doesn't seem to be the case here. Consider using the "Initialize" verb.